src/theme/lightTheme.js

```diff
@@ -24,7 +24,7 @@ const lightTheme = {
   "color-white": "#ffffff",
 
   "color-shadow-light": "rgba(0, 0, 0, 0.2)",
-  "font-family-text": '"Source Sans Pro"',
+  "font-family-text": "",
```

Comment:
No fonts are imported by default, provided by consumer.

Reply:
We can use a browser font as the default (e.g Helvetica or something) just something that looks pretty and close enough to Source Sans Pro. And then in our docz docs we provide the Source Sans Pro fonts (same as we are doing for politeiagui) since it is a Decred product and it needs to use the default Decred font, it's also one functional example more of how it should be used.